doc/coding-style-guide.md

```diff
@@ -195,7 +195,6 @@ package abc
 
 import (
     dynakubev1beta2  "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
-    dynakubev1beta3  "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
here its just an example how you would import different versions of the same api

doesn't really matter, you could replace it with 
```
    dynakubev1beta1  "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube"
    dynakubev1beta2  "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
```

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/3703/commits/0874d9c4a05488ef357ae4441e505dc134b1218d